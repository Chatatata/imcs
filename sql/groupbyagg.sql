select cs_group_max(Close,Day/3) from Quote_get('IBM');
select cs_group_min(Close,Day/3) from Quote_get('IBM');
select cs_group_sum(Close,Day/3) from Quote_get('IBM');
select cs_group_avg(Close,Day/3) from Quote_get('IBM');
select cs_group_var(Close,Day/3) from Quote_get('IBM');
select cs_group_dev(Close,Day/3) from Quote_get('IBM');
select cs_group_first(Close,Day/3) from Quote_get('IBM');
select cs_group_last(Close,Day/3) from Quote_get('IBM');
