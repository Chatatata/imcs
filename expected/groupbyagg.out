select cs_group_max(Close,Day/3) from Quote_get('IBM');
      cs_group_max       
-------------------------
 float4:{20.2,40.2,50.5}
(1 row)

select cs_group_min(Close,Day/3) from Quote_get('IBM');
      cs_group_min       
-------------------------
 float4:{10.5,30.2,50.5}
(1 row)

select cs_group_sum(Close,Day/3) from Quote_get('IBM');
      cs_group_sum       
-------------------------
 float8:{30.7,70.4,50.5}
(1 row)

select cs_group_avg(Close,Day/3) from Quote_get('IBM');
       cs_group_avg       
--------------------------
 float8:{15.35,35.2,50.5}
(1 row)

select cs_group_var(Close,Day/3) from Quote_get('IBM');
     cs_group_var      
-----------------------
 float8:{23.5225,25,0}
(1 row)

select cs_group_dev(Close,Day/3) from Quote_get('IBM');
   cs_group_dev    
-------------------
 float8:{4.85,5,0}
(1 row)

select cs_group_first(Close,Day/3) from Quote_get('IBM');
     cs_group_first      
-------------------------
 float4:{10.5,30.2,50.5}
(1 row)

select cs_group_last(Close,Day/3) from Quote_get('IBM');
      cs_group_last      
-------------------------
 float4:{20.2,40.2,50.5}
(1 row)

